docs/src/pages/[platform]/components/pagination/react.mdx

```diff
@@ -29,7 +29,7 @@ Import the Pagination component. To use Pagination as an uncontrolled component,
   <ExampleCode>
 ```jsx file=./examples/DefaultPaginationExample.tsx
 
-```
+````
```

Comment:
Why the quadruple quotes here?

Reply:
removed...IDK why my prettier likes it 😂